src/builder.rs

```diff
@@ -76,14 +76,14 @@ impl<'a> WebviewBuilder<'a> {
             w.set_title(title);
         }
 
-        if let Some(url) = self.url {
-            w.navigate(url);
-        }
-
         if let Some(init) = self.init {
             w.init(init);
         }
 
+        if let Some(url) = self.url {
```

Comment:
shouldn't this be called when running the webview instead of when building it? this actually explains why Tauri's __TAURI_INVOKE_HANDLER__ isn't working properly on Windows, since it's not going to the `init` callback before running the app.

Reply:
btw it works like a charm :) nice catch

Reply:
`run` is what event loop on each platform is going to do. I think it should be decoupled with other webview functions (internally) imho. This building process is more like inform the internal webview. It's webview module in winrt has more specific ordering issue.

Reply:
shouldn't this initialization code be run before the event loop, but after the builder builds the instance? 

an alternative would be exposing the bind function on the builder.

Reply:
The thing is when writing a binding closure you pretty much need a instance which we can't provide it yet during building builder. The builder is just a more idiomatic way to create webview instance. Users are still able to call other methods to keep building it.